articles/dns/tutorial-alias-pip.md

```diff
@@ -16,8 +16,8 @@ In this tutorial, you learn how to:
 
 > [!div class="checklist"]
 > * Create a network infrastructure.
-> * Create a web server virtual machine.
-> * Create an alias record.
+> * Create a web server virtual machine with a public IP.
+> * Create an alias record the points to the public IP.
```

Comment:
please replace "..record **the** points.." to "..record **that** points.."

Reply:
Sorry. Done.